python-main.js

```diff
@@ -694,44 +716,34 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
-
-        function formatHelpPanel(){
-            if($(".helpsupport_container").offset().left !== $("#command-help").offset().left && $(window).width() > 620){
-                $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
-                $(".helpsupport_container").css("left", $("#command-help").offset().left);
-            }
-            else if($(window).width() < 620){
-                $(".helpsupport_container").css("left", $("#command-help").offset().left - 200);
-                $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
-            }
-        };
-
+        $("#command-options").click(function (e) {
+            // Hide any other open menus and show/hide options menu
+            $('#helpsupport_container').addClass('hidden');
+            $('#options_container').toggleClass('hidden');
+            formatMenuContainer('command-options', 'options_container');
+            // Stop immediate closure
+            e.stopImmediatePropagation();
+        });
         $("#command-help").click(function (e) {
-            // Show help
-            formatHelpPanel();
-            // Toggle visibility
-            if($(".helpsupport_container").css("display") == "none"){
-                $(".helpsupport_container").css("display", "flex");
-                $(".helpsupport_container").css("display", "-ms-flexbox"); // IE10 support
-            } else {
-                $(".helpsupport_container").css("display", "none");
-            }
```

Comment:
Visibility is now toggled by adding/removing the `hidden` class to the `#helpsupport_container` and `#options_container` divs.

Reply:
Makes sense for consistency